src/Documentation/Documentation.v4.0.shfbproj

```diff
@@ -38,7 +38,7 @@
     <HelpTitle>openstack.net API Reference Documentation</HelpTitle>
     <ContentPlacement>AboveNamespaces</ContentPlacement>
     <RootNamespaceTitle>API Reference</RootNamespaceTitle>
-    <WebsiteSdkLinkType>Msdn</WebsiteSdkLinkType>
+    <WebsiteSdkLinkType>None</WebsiteSdkLinkType>
```

Comment:
The documentation projects are only built in the **Release** configuration; in addition, the **ReleaseNoDocs** configuration allows the release binaries to be built without documentation. Commit 512d41e105556d597d42fa21fd696b317f2e8191 leverages a new feature of SHFB (originally [requested here](https://shfb.codeplex.com/workitem/35487)) to reduce documentation build times as well. There are some requests like dotnet/roslyn#67 which could eventually result in substantial performance improvements for the `<inheritdoc/>` feature.

A minute doesn't seem like much of a savings, especially when you aren't building documentation for the debug builds (when users are iterating on new feature development).